Userland/Applications/Browser/WebDriverConnection.cpp

```diff
@@ -93,7 +93,7 @@ Messages::WebDriverSessionClient::GetNamedCookieResponse WebDriverConnection::ge
             }
         }
     }
-    return { {} };
+    return Optional<Web::Cookie::Cookie> {};
```

Comment:
...but this definitely isn't, we wouldn't want to return a valid IPC response if no one hooked up `on_get_cookies_entries`. Needs another return after the for loop instead, and keeping this as it is.

Reply:
Ill add the extra return above, correct.

Doing a `return { {} }` hangs in the IPC somewhere and never returns to the caller on the other side of the connection. With this at least we return (an empty Optional) even if something is horribly agoof.

Reply:
That seems like an unrelated problem that should not be papered over :)